### ISSUE TITLE
Beta Dashboard Holds a Lot of Open Connections

### ISSUE THREAD

**Issue by nebhale** [concourse/concourse#1806]
It appears that the beta dashboard in 3.6.0 is holding open a lot of persistent connections to the server.  In Safari, I commonly keep the dashboard open and then use it to open tabs for individual pages (typically failed builds).  When I open a couple of tabs beyond [my dashboard](https://java-experience.ci.springapps.io/beta/dashboard), I find that those tabs will not load.  It's not that they fail, it's that they are stalling as if they're waiting for a free connection to make a request on.  This is born out by the fact that closing the dashboard will *instantly* cause those pages to load.

My (admittedly amateur) guess is that browser has a limited number of simultaneous persistent connections to a given host and the dashboard is occupying most or all of them for dashboards of a large enough size.

I'd prefer to be able to open as many tabs against my server as possible, without them waiting on occupied connections.

**Comment by jama-pivotal** [comment]
Yup, its pretty janky right now @nebhale and does tend to spam the ATC. We did our best to make it not terrible, but it can definitely be optimized. We have a related issue https://github.com/concourse/concourse/issues/1729 that is scheduled to be worked on soon

**Comment by jama-pivotal** [comment]
We should keep an eye out for places where we poll without waiting for previous requests to finish

**Comment by mhuangpivotal** [comment]
The dashboard refreshes itself every 5 seconds. Every time it refreshes, it makes a new request for each pipeline. For a deployment with many teams/pipelines, the number of requests quickly add up. Here's our proposal:

- [ ] add `/api/v1/jobs` endpoint. This endpoint returns all visible jobs across all teams/pipelines.
- [ ] migrate the `groups` data to the jobs table to facilitate the above endpoint. Previously the `groups` was stored as a JSON config in the pipelines table.
- [ ] refactor the dashboard and the hd dashboard to use the new endpoint. This reduces the number of requests for each refresh to just a few (pipelines, jobs, user).
- [ ] do not make new requests if previous requests have not finished.
